config/initializers/devise.rb

```diff
@@ -9,7 +9,7 @@
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
   # config.secret_key = '79c8226c2c21085816562ba8fb57a483613dfcfb86920d25f4609f14f21d285fe60b4ca3e945cf72a61f16a369e19b15cd07fcde3c518c5871a70e120729df68'
-  
+  config.secret_key = '7b27c06d030597a433d1dc5feb5e728eb9d485acdec55438a3a79f9b7ce1cc006682ba7e1c027c4ca09adfcf15576e5388882101c1716b31f55dcce6f7b476a0'
```

Comment:
🤔 why not keep using the secret key base?

Reply:
this way you expose to everyone that can access the codebase the secret key that devise uses. It's safer to use an environment variable like the secret key

Reply:
So what would be the best aproach? Creating the env variable directly in the OS (ie: export my_env_var="secret") and just capturing it in my code like ENV['my_env_var'] or using configuration files containing the env (adding to .gitignore for production)?